src/interactive/details/history/graph_data.rs

```diff
@@ -42,6 +42,34 @@ impl Point {
     }
 }
 
+fn payload_string_as_f64(payload: &str) -> Option<f64> {
+    payload
+        .split(char::is_whitespace)
+        .find(|o| !o.is_empty())? // lazy trim
```

Comment:
Is `.trim_start().split(…).next()` or `.split(…).find(…)` nicer / cleaner / faster? :thinking:

Reply:
I think its readable enough and a bit shorter in the current way.